docs/developing.md

```diff
@@ -181,9 +181,7 @@ utilities (i.e. `pip`). Use an administrative command prompt where applicable
 for installing Python modules for all users.
 
 Pre-requisites:
-* Python 2.7 x86 with `pip`
-* Visual C++ for Python 2.7 development, available at:
-  <https://aka.ms/vcpython27>
+* Python 3.7.x x86 with `pip`
```

Comment:
only Python `3.7` or `>=3.7`?

Reply:
I mentioned 3.7 because that's what we have tested with. Since support for 3.7 is ending soon, we have plans to move on to 3.10+ in the near future at which point I think this line can be updated.